purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -407,7 +407,7 @@ class Purchases internal constructor(
     }
 
     /**
-     * Make a purchase.
+     * Make a purchase upgrading from a previous sku.
```

Comment:
I changed these since otherwise these 2 methods with the same name had almost the same documentation.